packages/landing/src/components/map.tsx

```diff
@@ -91,7 +91,7 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
   };
 
   updateVisibleLayers = (newLayers: string): void => {
-    if (!Config.map.visibleLayers) Config.map.visibleLayers = newLayers;
+    if (Config.map.visibleLayers === undefined) Config.map.visibleLayers = newLayers;
```

Comment:
== null is preferred, unless you really need to validate that this is undefined and not null

Reply:
All good, changed in new commit